src/Language/JavaScript/Dependencies.hs

```diff
@@ -0,0 +1,379 @@
+module Language.JavaScript.Dependencies
+  ( JSAST
+  , Dependencies
+  , jsAstDependencies
+  ) where
+
+import PacchettiBotti.Prelude
+
+import qualified Data.Char as Char
+import qualified Data.List as List
+import qualified Data.Set as Set
+import qualified Data.Text as Text
+import qualified Network.URI as URI
+import qualified Turtle
+
+import Data.Set (Set)
+import Language.JavaScript.Parser.AST
+
+type ModuleId = String
+type Dependencies = Set Text
+
```

Comment:
Could we have a docstring detailing what the purpose of this function is?

Reply:
I added a few comments.